src/containers/Tenants/Tenants.tsx

```diff
@@ -317,15 +314,14 @@ function getDBActionsColumn({
                     },
                 });
             }
-            if (clusterName && isDeleteDBAvailable && databaseName && databaseId) {
+            if (isDeleteDBAvailable) {
                 menuItems.push({
                     text: i18n('remove'),
                     iconStart: <TrashBin />,
                     action: () => {
                         uiFactory.onDeleteDB?.({
                             clusterName,
-                            databaseId,
-                            databaseName,
+                            databaseData: row,
```

Comment:
I want always show edit button in database actions similar to Edit, if some data is missing, there will be stub with an error